bottom-bar/src/main/java/com/roughike/bottombar/BottomBar.java

```diff
@@ -141,6 +143,11 @@ private void populateAttributes(Context context, AttributeSet attrs) {
             tabXmlResource = ta.getResourceId(R.styleable.BottomBar_bb_tabXmlResource, 0);
             isTabletMode = ta.getBoolean(R.styleable.BottomBar_bb_tabletMode, false);
             behaviors = ta.getInteger(R.styleable.BottomBar_bb_behavior, BEHAVIOR_NONE);
+
+            if (isIconsOnlyMode() && isShiftingMode()) {
```

Comment:
I just thought it looked a little weird, and would require a boolean, since a tab can have only one type because it's an enum. Also I think having a separate layout resource for the titleless tab was cleaner than reusing fixed / shifting tab layouts.

But yeah, I think this limitation could be removed, as there will probably be someone who would like have shifting tabs without titles.